packages/kit/test/apps/basics/test/cross-platform/client.test.js

```diff
@@ -452,6 +452,8 @@ test.describe('Scrolling', () => {
 
 		await page.goto('/anchor');
 		await page.evaluate(() => window.scrollTo(0, 1000));
+		// wait for the scroll to be processed before jumping to the next page
+		await page.waitForTimeout(1000);
```

Comment:
do we need a full second here or should we reduce it to 100ms or sth? Also is the same needed after the second scroll? A nicer way could be to await the fact that scrollY is 1000 :thinking:

Reply:
Perhaps you could use `page.waitForFunction` to avoid a timeout and instead wait for the page position to make it to 1000 pixels?